Month1/Week2/Section2/Month1-Week2-Section2/Spotify/Song.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Song.h"
+FSong::FSong()
+{
+    MinutesDuration = 0;
+    SecondsDuration = 0;
+    ArtistName = "";
+    SongName = "";
+}
+FSong::FSong(int EntryMinutesDuration, int EntrySecondsDuration, std::string EntryArtistName, std::string EntrySongName)
```

Comment:
Remember that parameters should be const if they are not modified inside the function. Please make this change for the whole review

Reply:
Ok, Understanded, and Added in all the program